### PR TITLE
Added option to set host name

### PR DESCRIPTION
**Pull request by kobihk**
Since the host name taken from the same string of ip/fqdn
we add the option to set host name.

### CODE REVIEW

File: roles/register_hypervisor/files/add_host [hunk 1]
```diff
@@ -23,36 +23,34 @@ def main():
...
+
+    parser.add_argument('--url', dest='url', required=True,
+                        help='the url for the Engine api')
+    parser.add_argument('--username', dest='username', required=True,
+                        help='the username for the Engine api')
+    parser.add_argument('--password', dest='password', required=True,
+                        help='the password for the Engine api')
+    parser.add_argument('--host', dest='host', required=True,
+                        help='the ip address or fqdn of the hypervisor host')
+    parser.add_argument('--host_name', dest='host_name', required=False,
```
Comment (fabianvf) [inline roles/register_hypervisor/files/add_host]: `--host-name` to maintain the pattern used by the rest of the cli utilities

File: roles/register_hypervisor/tasks/main.yml [hunk 2]
```diff
@@ -25,6 +25,7 @@
                                       --username='{{ engine_username }}'
                                       --password='{{ admin_password }}'
                                       --host='{{ inventory_hostname }}'
+                                      --host_name='{{ host_name }}'
```
Comment (fabianvf) [inline roles/register_hypervisor/tasks/main.yml]: Can you make this optional?  Like `{{ "--host_name=" + host_name if host_name is defined else ""}}` or something

File: roles/register_hypervisor/files/add_host (hunk 1 above)
Comment (kobihk) [inline roles/register_hypervisor/files/add_host]: Done!

File: roles/register_hypervisor/tasks/main.yml (hunk 2 above)
Comment (kobihk) [inline roles/register_hypervisor/tasks/main.yml]: Done!
The parse_args take care if the host-name is empty string:
`
    if not args.host_name:

        args.host_name = args.host
`

Comment (fabianvf) [inline roles/register_hypervisor/tasks/main.yml]: ah of course, thanks.

**Pull request merged**